### DISCUSSION TITLE
osu lazer reset to basic settings with no maps/skins

### DISCUSSION THREAD

**Discussion by crispychicken28** [ppy/osu#27039] [Q&A]
I was opening osu lazer and when I got in I saw I didn't have any maps and all my skins were gone. right before this I made a shortcut of the game and moved it to my desktop. I ended up deleting that shortcut since I didn't need it. though after I opened osu via the windows button and searching osu and clicking enter all my maps and stuff were gone. I afraid that somehow instead of deleting a shortcut I deleted osu entirely but yet it was still on my computer and can be opened. Here are some images describing it but its really all I have. 
sorry if this is in the wrong spot or category first time using github community 
![image](https://github.com/ppy/osu/assets/108493725/95781ae5-bb6b-4deb-af19-35ae3fdb6167)
![image](https://github.com/ppy/osu/assets/108493725/7197bcce-c270-47d8-9872-9f3a3a0a744f)
![osu_2024-01-26_20-50-31](https://github.com/ppy/osu/assets/108493725/7f7078b8-6320-4c9d-8cea-0e8e4731e592)
![image](https://github.com/ppy/osu/assets/108493725/0db670e6-d560-49f6-930f-7442eb787475)
sorry if this is in the wrong spot or category its my first time asking the github community.

**Comment by crispychicken28** [comment]
![image](https://github.com/ppy/osu/assets/108493725/98d9ea86-21c0-4ecb-8c36-f21e01652be0)
heres a screenshot of what I had on my desktop in which i deleated the far most right one

**Comment by Rekunan** [comment]
Can you try seeing if changing the folder location to `%AppData%/osu` works?

**Reply by crispychicken28** [reply]
When I did this it reverted back to first launch screen again for some reason.

I did somehow manage to transfer my old OSU standard progress over to OSU lazer which is one of the things I tried to do in the past so I'm pretty satisfied with leaving it like this.

(sorry for the late response I've been busy with school and forgot about this)

**Comment by peppy** [comment]
You're likely running a shortcut pointing to an old version. Delete the shortcut and run lazer from the start menu, or re-run the installer to start the correct version.

If this doesn't work, you may need to manually restore a previous version of `client.realm` in the osu! data folder.

**Reply by crispychicken28** [reply]
for some reason I must have ran the installer multiple times in the past because I have multiple different OSU lazer location in different spots and none of them seem to be shortcuts. I'm not the best with computer files and stuff behind the games so these could all (and probably) lead to the same game location. (I did delete that old shortcut.)

I did somehow manage to transfer my old OSU standard progress over to OSU lazer which is one of the things I tried to do in the past so I'm satisfied with leaving it like this with that data.

When i re-ran the installer it started up with ^ this data instead of the past OSU lazer data. (I did change the file location to OSU standard so I'm guessing its using that instead of the old lost OSU lazer progress.

(sorry for the late response I've been busy with school and forgot about this)
